packages/ckeditor5-image/src/imageresize/imageresizeediting.js

```diff
@@ -120,7 +122,7 @@ export default class ImageResizeEditing extends Plugin {
 		editor.conversion.for( 'upcast' )
 			.attributeToAttribute( {
 				view: {
-					name: 'figure',
+					name: imageType === 'image' ? 'figure' : 'span',
```

Comment:
This conversion is incorrect. In the case of inline images, the upcasted data will be `<img />` but not `<span><img /></span>` (see my comment in tests).